sddi/Dockerfile

```diff
@@ -1,107 +1,330 @@
+ARG CKANEXT_GEOVIEW_VERSION="v0.2.2"
+ENV CKANEXT_GEOVIEW_VERSION=${CKANEXT_GEOVIEW_VERSION}
+ENV CKANEXT_GEOVIEW_GITHUB_URL="https://github.com/ckan/ckanext-geoview"
+
+RUN set -ex && \
+    curl -o /wheels/ckanext-geoview-dev-requirements.txt \
+        ${CKANEXT_GEOVIEW_GITHUB_URL}/raw/${CKANEXT_GEOVIEW_VERSION}/dev-requirements.txt && \
+    pip install -r /wheels/ckanext-geoview-dev-requirements.txt && \
+    pip wheel --wheel-dir=/wheels \
+        git+${CKANEXT_GEOVIEW_GITHUB_URL}.git@${CKANEXT_GEOVIEW_VERSION}#egg=ckanext-geoview
+
+# ckanext-theme-sddi #############################################################
+ARG CKANEXT_THEME_SDDI_VERSION="0.0.3"
+ENV CKANEXT_THEME_SDDI_VERSION=${CKANEXT_THEME_SDDI_VERSION}
+ENV CKANEXT_THEME_SDDI_GITHUB_URL="https://github.com/MarijaKnezevic/ckanext-theme-sddi"
+
+RUN set -ex && \
+    curl -o /wheels/ckanext-theme-sddi-dev-requirements.txt \
+        ${CKANEXT_THEME_SDDI_GITHUB_URL}/raw/${CKANEXT_THEME_SDDI_VERSION}/requirements.txt && \
+    pip install -r /wheels/ckanext-theme-sddi-dev-requirements.txt && \
+    pip wheel --wheel-dir=/wheels \
+        git+${CKANEXT_THEME_SDDI_GITHUB_URL}.git@${CKANEXT_THEME_SDDI_VERSION}#egg=ckanext-theme-sddi
+
+# ckanext-clamav #############################################################
+ARG CKANEXT_CLAMAV_VERSION="a1d23ac"
+ENV CKANEXT_CLAMAV_VERSION=${CKANEXT_CLAMAV_VERSION}
+ENV CKANEXT_CLAMAV_GITHUB_URL="https://github.com/DataShades/ckanext-clamav"
+
+RUN set -ex && \
+    curl -o /wheels/ckanext-clamav-requirements.txt \
+        ${CKANEXT_CLAMAV_GITHUB_URL}/raw/${CKANEXT_CLAMAV_VERSION}/requirements.txt && \
+    pip install -r /wheels/ckanext-clamav-requirements.txt && \
+    pip wheel --wheel-dir=/wheels \
+        git+${CKANEXT_CLAMAV_GITHUB_URL}.git@${CKANEXT_CLAMAV_VERSION}#egg=ckanext-clamav
+
+# ckanext-dcat ##########################################################
+ARG CKANEXT_DCAT_VERSION="v1.5.1"
+ENV CKANEXT_DCAT_VERSION=${CKANEXT_DCAT_VERSION}
+ENV CKANEXT_DCAT_GITHUB_URL="https://github.com/ckan/ckanext-dcat"
+
+RUN set -ex && \
+    curl -o /wheels/ckanext-dcat-requirements.txt \
+        https://raw.githubusercontent.com/ckan/ckanext-dcat/${CKANEXT_DCAT_VERSION}/requirements.txt && \
+    pip install -r /wheels/ckanext-dcat-requirements.txt && \
+    pip wheel --wheel-dir=/wheels \
+        git+${CKANEXT_DCAT_GITHUB_URL}.git@${CKANEXT_DCAT_VERSION}#egg=ckanext-dcat
+
+# ckanext-security ######################################################
+ARG CKANEXT_SECURITY_VERSION="5724341"
+ENV CKANEXT_SECURITY_VERSION=${CKANEXT_SECURITY_VERSION}
+ENV CKANEXT_SECURITY_GITHUB_URL="https://github.com/MarijaKnezevic/ckanext-security"
     
-# #############################################################################
-# # Runtime stage
-# #############################################################################
-FROM ${BASEIMAGE_REPOSITORY}:${BASEIMAGE_VERSION} as runtime
```

Comment:
The `sddi` image needs use our `base` image as base.
@MarijaKnezevic Discuss with @aleksandralazoroska @ilchebedelovski which extension go in which Dockerfile.

Reply:
The extensions are solved, part of them are in sddi-base and the rest of them are in sddi according to the documentation in the Readme file.